admin-dev/themes/new-theme/js/components/form/submittable-input.js

```diff
@@ -77,21 +78,34 @@ export default class SubmittableInput {
    * @private
    */
   submitInput(e) {
```

Comment:
But I don't understand this one, seems like there can be two types of parameters here, the input OR the button (since it's related to click and keyup event) But it seems like the `findInput` function expects it to be a button It's not very clear

Reply:
Fixed it, the submitInput on keyup wasn't working at all